oshi-core/src/main/java/oshi/driver/mac/ThreadInfo.java

```diff
@@ -59,8 +59,8 @@ public static List<ThreadStats> queryTaskThreads(int pid) {
         List<ThreadStats> taskThreads = new ArrayList<>();
         // Only way to get thread info without root permissions
         // Using the M switch gives all threads with no possibility to filter
-        List<String> psThread = ExecutingCommand.runNative("ps -awwxM").stream().filter(s -> s.contains(pidStr))
-                .collect(Collectors.toList());
+        List<String> psThread = ExecutingCommand.runNative("ps -awwxM").stream().parallel()
+                .filter(s -> s.contains(pidStr)).collect(Collectors.toList());
```

Comment:
Probably not needed here but doesn't hurt, we're just filtering a list of strings and you'd really have to benchmark this to see if the parallel thread overhead beats the savings from simultaneously doing hundreds of `String.contains()` checks (which are pretty fast).